web/forms.py

```
@@ -832,38 +833,96 @@ def clean_topics(self):
     )
 
+    def clean_course_title(self):
+        """Validate and clean the course_title field."""
+        title = self.cleaned_data.get("course_title")
+        # Validate title contains only valid characters
+        if not re.match(r"^[\w\s-]+$", title):
+            raise forms.ValidationError("Title can only contain letters, numbers, spaces, and hyphens")
+
+        return title
+
+    def clean_preferred_session_times(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean_preferred_session_times`

(ANN201)